ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/ListAuthenticatedUserRightsAuthIT.scala

```diff
@@ -16,14 +16,15 @@ class ListAuthenticatedUserRightsAuthIT extends ServiceCallAuthTests {
   override def serviceCallName: String =
     "UserManagementService#ListUserRights(<authenticated-user>)"
 
-  override def serviceCallWithToken(token: Option[String]): Future[Any] =
-    stub(UserManagementServiceGrpc.stub(channel), token).listUserRights(ListUserRightsRequest())
+  protected def serviceCall(context: ServiceCallContext): Future[Any] =
+    stub(UserManagementServiceGrpc.stub(channel), context.token)
+      .listUserRights(ListUserRightsRequest())
```

Comment:
Aren't we supposed to populate the idp field?

Reply:
well, spotted, though currently there is no test to try this out, it's not part of this PR yet :)